pymatbridge/matlab_magic.py

```diff
@@ -211,6 +211,8 @@ def matlab(self, line, cell=None, local_ns=None):
             else:
                 e_s = "There was an error running the code:\n %s"%code
                 result_dict = self.eval(code)
+        except MatlabInterperterError:
+            raise
         except:
```

Comment:
We came across this before in #79, and we couldn't figure out what exception was meant to go there. It might make sense to just capture all exceptions from `zmq` (e.g. wrapped in `MatlabCommunicationError` or something like that) and catch just those here.